Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Historical Plagues

## Changelog:
- The Plague starts in Andorhal at the beginning of the year 603 bookmark.
  - Alexei, Illucia, Jandice Barovs, and Pharamond Rivendare convert to the Scourge faith.
- If you play from the year 583 bookmark, the Scourge AI starts a Plague scheme after 6 years of ruling.
- Lordaeron cities get more development.
- The Plague spawns 75% more troops.
- Calm and Patient characters would like to join the Plague scheme.

# Tasks:
- [x] The Scourge should check for a few kingdoms away


## Tests:


# How to test:
Test if features work in the year 603 bookmark.

Start in the year 583 bookmark, use `event WCCSC.1` to spawn the Scourge. Confirm the Scourge AI opens a Plague scheme.
